Code/Framework/AzCore/AzCore/IO/Streamer/Scheduler.cpp

```diff
@@ -122,13 +122,60 @@ namespace AZ::IO
         }
     }
 
+    bool Scheduler::IsSuspended() const
+    {
+        return m_isSuspended;
+    }
+
     void Scheduler::CollectStatistics(AZStd::vector<Statistic>& statistics)
     {
+        statistics.push_back(Statistic::CreateBoolean(
+            SchedulerName, "Is suspended", m_isSuspended,
+            "Whether or not the scheduler is suspended. When suspended the scheduler will not do any processing and effectively prevents "
+            "Streamer from doing any work.", Statistic::GraphType::None));
 #if AZ_STREAMER_ADD_EXTRA_PROFILING_INFO
-        statistics.push_back(Statistic::CreateInteger(SchedulerName, "Is idle", m_stackStatus.m_isIdle ? 1 : 0));
-        statistics.push_back(Statistic::CreateInteger(SchedulerName, "Available slots", m_stackStatus.m_numAvailableSlots));
-        statistics.push_back(Statistic::CreateFloat(SchedulerName, "Processing speed (avg. mbps)", m_processingSpeedStat.CalculateAverage()));
-        statistics.push_back(Statistic::CreatePercentage(SchedulerName, ImmediateReadsName, m_immediateReadsPercentageStat.GetAverage()));
+        statistics.push_back(Statistic::CreateBoolean(
+            SchedulerName, "Is idle", m_stackStatus.m_isIdle,
+            "Whether or not Streamer is scheduling requests. If this is not true most of the time, it indicates scheduling is too expensive, "
+            "that there are nodes that take too long and/or there are callbacks that take  too long. Note that some nodes are not "
```

Comment:
whitespace: There is an additional space between "take" and "too"
```suggestion
            "that there are nodes that take too long and/or there are callbacks that take too long. Note that some nodes are not "
```

Reply:
Fixed in next commit.